src/main/java/com/sw19/sofa/domain/ai/controller/AiSummaryController.java

```diff
@@ -0,0 +1,23 @@
+package com.sw19.sofa.domain.ai.controller;
+
+import com.sw19.sofa.domain.ai.service.AiSummaryService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/ai")
```

Comment:
```suggestion
@RequestMapping("/ai")
``` 
로 변경하는게 더 통일성이 있어보입니다

Reply:
controller는 제가 응답test용으로 만든거라 추후 삭제할 예정입니다.

Reply:
아하 그렇군요!